Integrations/DomainTools_Iris/iris.py

```diff
@@ -0,0 +1,393 @@
+from CommonServerPython import *
+import requests
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS '''
+
+BASE_URL = 'http://api.domaintools.com'
+USERNAME = demisto.params().get('username')
+API_KEY = demisto.params().get('apikey')
+RISK_THRESHOLD = int(demisto.params().get('risk_threshold'))
+PIVOT_COUNT_THRESHOLD = int(demisto.params().get('pivot_count_threshold'))
+YOUNG_DOMAIN_TIMEFRAME = int(demisto.params().get('young_domain_timeframe'))
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, path, other_params=None):
+    """
+    HTTP request helper function
+    """
+    params = {'app_partner': 'demisto', 'app_name': 'Iris Plugin', 'app_version': '1', 'api_username': USERNAME,
+              'api_key': API_KEY}
+    if other_params:
+        params.update(other_params)
+    url = '{}{}'.format(BASE_URL, path)
+    res = requests.request(
+        method=method,
+        url=url,
+        params=params
+    )
+    res_json = res.json()
+    if not res.ok:
+        txt = 'error in URL {} status code: {} reason: {}'.format(url, res.status_code,
+                                                                  res_json['error'].get('message'))
+        demisto.error(txt)
+        raise Exception(txt)
+
+    try:
+        return res_json.get('response')
+    except Exception as ex:
+        demisto.debug(str(ex))
+        demisto.results({"Type": entryTypes["error"], "ContentsFormat": formats["text"], "Contents": res.text})
+
+
+def find_age(create_date):
+    time_diff = datetime.now() - datetime.strptime(create_date, "%Y-%m-%d")
+    return time_diff.days
+
+
+def get_threat_component(components, threat_type):
+    for component in components:
+        if component['name'] == threat_type:
+            return component
+    else:
+        return None
+
+
+def convert_empty_to_null(data_obj):
+    # Check for empty lists, dicts, and strs. Set to null if found to be empty.
+    if isinstance(data_obj, dict) and (data_obj is None or len(data_obj) != 0):
+        for k, v in data_obj.items():
+            if isinstance(data_obj[k], dict) or isinstance(data_obj[k], list):
+                convert_empty_to_null(data_obj[k])
+            if not isinstance(v, int) and (v is None or len(v) == 0):
+                data_obj[k] = None
+    elif isinstance(data_obj, list) and (data_obj is None or len(data_obj) != 0):
+        for index, item in enumerate(data_obj):
+            convert_empty_to_null(item)
+            if not isinstance(item, int) and (item is None or len(item) == 0):
+                data_obj[index] = None
+
+
+def create_context_output(domain_result):
```

Comment:
Please change to more specific function name, and not so generic one, something like `create_domain_context_outputs`